filebeat/module/logstash/log/config/log.yml

```diff
@@ -5,12 +5,10 @@ paths:
 {{ end }}
 exclude_files: [".gz$"]
 
-{{ if eq .format "plain" }}
 multiline:
-  pattern: ^\[[0-9]{4}-[0-9]{2}-[0-9]{2}
+  pattern: ^(\[[0-9]{4}-[0-9]{2}-[0-9]{2}|{)
```

Comment:
Is it possible to trick this pattern with `{` character? I mean having a plain text file with curly brackets inside.

Reply:
Yes, it's a good point. If there's a multiline log event where, say, the 2nd line starts with a `{`, then this pattern breaks down. Unfortunately, I'm not really sure how to handle this scenario well.

Reply:
How about extending this pattern to `{"level"` ?

Reply:
For JSON-formatted logs, each log line is a JSON object. Being an _object_, I don't want to depend on a specific property, e.g. `level`, being the first one.